spec/views/users/index.html.erb_spec.rb

```diff
@@ -1,5 +1,21 @@
 require 'rails_helper'
 
-RSpec.describe 'users/index.html.erb', type: :view do
-  #  pending "add some examples to (or delete) #{__FILE__}"
+RSpec.feature 'User Index Page' do
+  scenario 'Viewing user list' do
+    # Create test users directly in the test database
+    user1 = User.create(name: 'User One', bio: 'Bio One')
+    user2 = User.create(name: 'User Two', bio: 'Bio Two')
+    user3 = User.create(name: 'User Three', bio: 'Bio Three')
+
```

Comment:
- [ ] It would be better if we implement test cases for the following
![Screenshot from 2023-09-20 16-10-01](https://github.com/Gardimy/BlogApp/assets/46388113/715156e0-e918-45db-863c-3125ba57754a)